addon/components/emoji-picker.js

```diff
@@ -5,12 +5,26 @@ import {debounce, next, throttle} from 'ember-runloop';
 import layout from '../templates/components/emoji-picker';
 import service from 'ember-service/inject';
 import observer from 'ember-metal/observer';
-import get from 'ember-metal/get';
 import ClickOutsideMixin from 'ember-click-outside/mixins/click-outside';
-
 import {A} from 'ember-array/utils';
 import {default as EObject} from 'ember-object';
-const  O = EObject.create.bind(EObject);
+import {EMOJI_CATEGORIES_ARRAY, EMOJI_TONES_ARRAY} from "../utils/constants";
+
+const O = EObject.create.bind(EObject);
+const EMOJI_PICKER_SCROLLABLE_ELEMENT = '.eeo-emojiPicker-scrollable';
+const DEPENDENT_KEYS_FOR_EMOJI_CATEGORIES = EMOJI_CATEGORIES_ARRAY.join(',');
+const DEPENDENT_KEYS_FOR_EMOJI_SERVICE = (() => {
+  const dependentKeys = [];
+
+  EMOJI_CATEGORIES_ARRAY.forEach(category => {
+    EMOJI_TONES_ARRAY.forEach(tone => {
+      const key = `emojiService.${category}__tone_${tone}`;
+      dependentKeys.push(key);
+    });
+  });
+
+  return dependentKeys.join(',');
+})();
```

Comment:
Not sure, I actually had them there initially but moved them as they're only used in this certain component.